### PR TITLE
Add improvements to application tutorial

### PR DESCRIPTION
**Pull request by mattj-io**
## Description
https://jira.mesosphere.com/browse/DOCS-2037

## Urgency
- [ ] Blocker 
- [ ] High
- [ x] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

**Comment by mattj-io** [comment]
Review request pls oss-team

File: 1.10/tutorials/dcos-101/app1.md [hunk 1]
```diff
@@ -10,30 +10,30 @@ menu_order: 3
 <table class="table" bgcolor="#FAFAFA"> <tr> <td style="border-left: thin solid; border-top: thin solid; border-bottom: thin solid;border-right: thin solid;"><b>Important:</b> Mesosphere does not support this tutorial, associated scripts, or commands, which are provided without warranty of any kind. The purpose of this tutorial is to demonstrate capabilities, and may not be suited for use in a production environment. Before using a similar solution in your environment, you must adapt, validate, and test.</td> </tr> </table>
 
 # Objective
-You now have a working persistence layer -redis- running in your cluster.
-In this section you deploy the first app connecting to redis. Note that this tutorial deliberately choose to focus on the principles and to deploy a very simple app with no further logic than connecting to redis.
+You now have a working persistence layer - [Redis](https://redislabs.com/) - running in your cluster.
```
Comment (pleia2) [inline 1.10/tutorials/dcos-101/app1.md]: Confirmed via section 4d of the Redis trademark guidelines that we want this capitalization. Line 8 also has a redis reference that should be capitalized.

File: 1.10/tutorials/dcos-101/app1.md [hunk 2]
```diff
@@ -10,30 +10,30 @@ menu_order: 3
...
-  * Have a look at the [app definition](https://raw.githubusercontent.com/joerg84/dcos-101/master/app1/app1.json). This app definition will download the python script and then run it inside the `mesosphere/dcos-101` Docker container.
-  * Add app to Marathon: `dcos marathon app add https://raw.githubusercontent.com/joerg84/dcos-101/master/app1/app1.json`
-* You have multiple options to check that app1 is running:
+* Review app
+  * Let us first take a short look at the [app](https://raw.githubusercontent.com/joerg84/dcos-101/master/app1/app1.py). It is very simple and just checks whether it can reach Redis and then prints the total number of keys stored there.
+* Deploy app
+  * The python script has a dependency on the [redis-python](https://pypi.python.org/pypi/redis) library, which you cannot assume to be present on all agent nodes. Because of this, you should run it in the `mesosphere/dcos-101` Docker container that provides all of the dependencies. Feel free to take a look at the [DOCKERFILE](https://github.com/joerg84/dcos-101/blob/master/app1/DOCKERFILE), which was used to create the `mesosphere/dcos-101` image.
```
Comment (pleia2) [inline 1.10/tutorials/dcos-101/app1.md]: Not sure why we're calling it "redis-python" in the link (is this a package name on mac or something?), elsewhere I've seen it referred to as redis-py or just the "Python client for Redis"

File: 1.10/tutorials/dcos-101/app1.md [hunk 3]
```diff
@@ -10,30 +10,30 @@ menu_order: 3
...
-Marathon is also referred to as the init system of DC/OS, as its main job to support long running services.
+You have just deployed your first app using [Marathon](https://mesosphere.github.io/marathon/) directly. Also note that the Redis service itself is running via Marathon.
+Marathon is referred to as the init system of DC/OS, as its main job is to support long running services.
 Marathon also allows for scaling or uninstalling of applications.
-As such you have multiple options to deploy and maintain apps on Marathon besides the DC/OS GUI:
+There are multiple options to deploy and maintain apps on Marathon besides the DC/OS GUI:
 
-* DC/OS CLI: You have just used that option to deploy your app1. To get more information check `dcos marathon app --help`.
-* HTTP endpoints: Marathon also comes with an extensive [REST API](https://mesosphere.github.io/marathon/docs/generated/api.html) which can also be used to deploy apps
+* DC/OS CLI: You have just used this option to deploy your app1. To get more information on the marathon CLI use `dcos marathon app --help`.
```
Comment (pleia2) [inline 1.10/tutorials/dcos-101/app1.md]: With changes above, there's now some inconsistency with how we refer to the app ("app" vs. "app1"). Here I'd say either "your app" or simply "app1".

**Comment by mattj-io** [comment]
Changes made as suggested.

**Pull request merged**